core/dbt/context/common.py

```diff
@@ -329,6 +329,22 @@ def get_datetime_module_context():
     }
 
 
+def get_context_modules():
+    return {
+        'pytz': get_pytz_module_context(),
+        'datetime': get_datetime_module_context(),
+    }
+
+
+def generate_config_context(cli_vars):
+    context = {
+        'env_var': env_var,
+        'modules': get_context_modules(),
+    }
+    context['var'] = Var(None, context, cli_vars)
```

Comment:
This is probably one of those things we'll want to consider when describing (and overhauling) the compilation context -- it's kind of wild that we need to provide a context to build the context... Obviously ok for now though